### DISCUSSION TITLE
How can i insert 'raw data' on 'attribute'?

### DISCUSSION THREAD

**Discussion by joon14-lee** [liftoff-sr/CIPster#24] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/83627718/150730026-46048a9c-c501-4dfc-882b-132091150665.png)

Hello,

It proceeded normally until compiling and executing CIPSter's sample program. However, when I saw raw data in 'attribute' by adding an instance in the scanner program, I confirmed that there was no 128 bytes of data I wanted to create. 
When using the C-based OpENER, I confirmed that 'raw data' was created without any problems.

How can I make "raw data" on "attribute"?

Additionally, I saw the addition of Instance within the CreateAssembleInstance() function of the sample program, but I couldn't see the addition of Attribute.

I would like to ask you about how to make doxygen documents.

**Comment by resetpointer** [comment, marked as answer]
This issue was addressed in the below commit, would you please give it a try.

https://github.com/liftoff-sr/CIPster/commit/4fca62756f4e3f742a6dbd5bd5a046ae03f11e02
